Tools/clinic/clinic.py

```diff
@@ -2593,11 +2593,10 @@ def __repr__(self) -> str:
 def parse_file(
         filename: str,
         *,
-        ns: argparse.Namespace,
         output: str | None = None,
+        verify: bool = True,
+        limited_capi: bool = DEFAULT_LIMITED_CAPI,
```

Comment:
I would prefer if it specify the version of the limited API, not a simple boolean.

Reply:
For now, it only supports Python 3.13. The value can be changed to an int later